wagtail/contrib/wagtailsitemaps/sitemap_generator.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, site=None):
         self.site = site
 
     def location(self, obj):
-        return obj.full_url
+        return obj.url
```

Comment:
Hi @leviadler - I think this change is doing the opposite of what you want...? `obj.full_url` will always give the full URL including domain - `obj.url` will return a local URL on single-site Wagtail installations.